compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -924,7 +924,7 @@ impl<D: Deps> DepGraphData<D> {
             // Promote the previous diagnostics to the current session.
             qcx.store_side_effects(dep_node_index, side_effects.clone());
 
-            let handle = qcx.dep_context().sess().diagnostic();
+            let handle = qcx.dep_context().sess().dcx();
```

Comment:
```suggestion
            let dcx = qcx.dep_context().sess().dcx();
```

I assume `handle` was meant to be `handler`.

Reply:
Good catch. I did lots of searching with regexps to find as many of the old names as I could, but I didn't look for `handle`. Looks like this was the only place it was used for a `Handler`, and presumably it was a typo. I have updated the code to change this one.